File: C++/First Steps in Coding - Basics of C++/While Loops/Exercise/Old Books.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	int checkedBooks=0;
	string desiredBook;
	getline(cin, desiredBook);
	while (true) {
		string bookName;
		getline(cin, bookName);
		if (bookName == "No More Books") {
			cout << "The book you search is not here!\nYou checked " << checkedBooks << " books.";
			break;
		}
		if (bookName == desiredBook) {
			cout << "You checked " << checkedBooks << " books and found it.";
			break;
		}
		else {
			checkedBooks += 1;
			continue;
		}
	}
}

File: Python/Basics/While Loop/Exercise/OldBooks.py
book = input()
c = 0
while True:
    bookTest = input()

    if bookTest == book:
        print(f"You checked {c} books and found it.")
        break

    elif bookTest == "No More Books":
        print(f"The book you search is not here!\nYou checked {c} books.")
        break

    else:
        c+= 1